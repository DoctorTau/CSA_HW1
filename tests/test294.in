229
4152 3036 -7501 -6712 -9802 7136 -4481 2361 -2932 -7008 8876 -1000 -3109 9512 -4577 -4701 1530 -4196 -6693 6273 -1469 -5263 -9109 2566 2663 -1174 4811 -3219 -4041 3342 7576 -672 1521 6535 4188 7536 473 5838 4196 4002 -4319 -8144 -6255 -586 -7605 2891 -2842 2232 -7734 -9212 4279 7166 2706 4857 7000 -175 -2098 -8015 7596 599 -9129 -3183 -2006 -6046 -5466 -7108 6557 7930 -9602 9826 6820 4619 -8010 -9814 -5471 5764 -7257 4704 3338 -4435 2606 2359 -8484 5995 -2181 -1438 256 4861 -6640 -5169 2994 -5113 6774 -4102 3817 3961 -111 -4731 -8454 326 -9474 9464 -4164 8372 7580 -9054 3995 8613 -3666 -1838 1478 833 -2791 -5076 8829 -9290 7806 2234 8394 6917 2860 4195 8366 5917 -7162 -8204 -9196 6002 3515 -2813 -5264 1870 1880 -1241 -2692 8136 -8861 260 -6538 9144 -6301 4874 8137 -320 5502 -2503 -4780 -7870 1638 -6845 5059 -5164 4439 -2678 4799 6243 5940 -9210 -5022 8860 -4851 3440 2656 -7322 -3712 6756 6573 -5637 -7990 7147 -1899 -1252 -1339 -2112 -8706 4864 -6922 -2452 1197 -6633 8086 -6410 -2735 1445 2631 -8883 4666 -1319 -4778 -8601 -6648 -6337 7399 9764 -7752 9702 9845 -5958 1201 -8888 8680 -443 3385 3873 -1319 4338 793 -1416 6500 6472 6926 2217 -4081 9450 -7292 -8449 5661 6612 7751 669 469 -5522 3982 7206 2910 -1459 932 6060 3464 