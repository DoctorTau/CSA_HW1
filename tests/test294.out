-6712 -9802 -2932 -7008 -1000 -4196 -1174 -672 -8144 -586 -2842 -7734 -9212 -2098 -2006 -6046 -5466 -7108 -9602 -8010 -9814 -8484 -1438 -6640 -4102 -8454 -9474 -4164 -9054 -3666 -1838 -5076 -9290 -7162 -8204 -9196 -5264 -2692 -6538 -320 -4780 -7870 -5164 -2678 -9210 -5022 -7322 -3712 -7990 -1252 -2112 -8706 -6922 -2452 -6410 -4778 -6648 -7752 -5958 -8888 -1416 -7292 -5522 